dotrix_pbr/src/material.rs

```diff
@@ -4,6 +4,9 @@ use dotrix_core::renderer::UniformBuffer;
 use dotrix_core::{Assets, Color, Id, Renderer};
 
 const DUMMY_TEXTURE: &str = "dotrix::dummy_texture";
+const DUMMY_ROUGHNESS_TEXTURE: &str = "dotrix::dummy_roughness_texture";
```

Comment:
Maybe we should rollback to a single dummy texture now, what do you think?

Reply:
Yeah we can try it

Reply:
It's done, maybe you could have a quick check

Reply:
Looks good and works on my linux laptop